src/tools/replay.cpp

```diff
@@ -156,7 +156,10 @@ Element& pop_next_action(std::string expected_action_type) {
 		replay_fps_limit->frame_finished();
 	}
 	
-	last_action_type = to_return->Value();
+	// control_click actions are not meaningful for debugging
```

Comment:
What is this?

Reply:
Tracking the last action to be replayed, which is helpful in the error messages for incorrect advance_time behavior: you want to know which command replayed last (that's the incorrect one)

Reply:
That wasn't the question. I'm asking what `control_click` is.

Reply:
`control_click` is an action recorded by a dialog, which means a control with the given id got triggered. (Keystrokes  that trigger controls' click events also record this event.)

They're not meaningful as a last_action_type because they get picked up and processed by the dialog's internal event loop when it runs.

Reply:
Ah, it's a poor choice of name, I guess. It sounds like holding the control key and pressing a mouse button. Would probably be less confusing as `click_control`.